packages/loader/container-loader/src/containerContext.ts

```diff
@@ -240,10 +240,18 @@ export class ContainerContext implements IContainerContext {
         this.runtime.processSignal(message, local);
     }
 
+    /**
+     * @deprecated - use getRuntimeEntryPoint instead
+     */
     public async request(path: IRequest): Promise<IResponse> {
         return this.runtime.request(path);
     }
 
+    public async getRuntimeEntryPoint(): Promise<IFluidObject> {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-return
+        return (this.runtime as any).getEntryPoint?.();
```

Comment:
i didn't bother updating definitions to just using a cast in the prototype to access the entry point